scripts/diagnose-network.sh

```
@@ -0,0 +1,139 @@
+    echo -e "\033[33m[WARNING] $@\033[0m"
+}
+
+# --- Pre-flight Checks ---
+info "Starting network diagnostics..."
+
+if [ ! -f "$ENV_FILE" ]; then
+    error ".env file not found. Cannot proceed without Supabase configuration."
+    exit 1
+fi
+
+source "$ENV_FILE"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)